src/main/client/scene/game-scene.ts

```diff
@@ -43,7 +43,7 @@ export class GameScene extends Scene {
          this.terrainTexture = this.textures.addCanvas('terrain', canvas);
          this.add.sprite(0, 0, 'terrain');
       });
-      this.mapComponent.updated$.subscribe(() => this.terrainTexture.update());
+      this.mapComponent.updated$.subscribe(() => this.terrainTexture && this.terrainTexture.update());
```

Comment:
Just a shorter version of this haha:
if(this.terrainTexture){
   this.terrainTexture.update()
}